src/Pages/form/Components/ArrivalOnset.tsx

```diff
@@ -0,0 +1,408 @@
+import "date-fns";
```

Comment:
The line of code is around 400 lines. it is quite hard to debug. So, please extract it into 2 or more components.